monkey/infection_monkey/control.py

```diff
@@ -63,37 +67,66 @@ def wakeup(self, parent=None):
             timeout=MEDIUM_REQUEST_TIMEOUT,
         )
 
-    def find_server(self, default_tunnel=None):
-        logger.debug(f"Trying to wake up with Monkey Island server: {self.server_address}")
-        if default_tunnel:
-            logger.debug("default_tunnel: %s" % (default_tunnel,))
+    def find_server(self, servers: Sequence[str]):
```

Comment:
Move this out of `control.py` to a better home. For now, I think it can just go in `monkey.py` or it can be a utility function somewhere in `infection_monkey.network.relay`.